src/components/editor/WysiwygEditor.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+import { EditorState } from "draft-js";
+import { Editor } from "react-draft-wysiwyg";
+import "react-draft-wysiwyg/dist/react-draft-wysiwyg.css";
+
+class WysiwygEditor extends Component {
```

Comment:
you can use react quill package as well, then you don't have to write class component